src/AppInstallerCLICore/Workflows/WorkflowBase.cpp

```diff
@@ -710,6 +710,7 @@ namespace AppInstaller::CLI::Workflow
         }
 
         Logging::SetExecutionStage(static_cast<uint32_t>(context.Get<Execution::Data::ExecutionStage>()));

```

Comment:
Move this all to the context method.  Remove the Data and just use the member.

Reply:
Have to still use Data due to clash between Data::ExecutionStage and WorkFlow::ExecutionStage

Reply:
? I mean remove the Data item completely.